fdbserver/DataDistribution.actor.cpp

```diff
@@ -4313,13 +4314,18 @@ ACTOR Future<Void> dataDistribution(Reference<DataDistributorData> self)
 				anyZeroHealthyTeams = zeroHealthyTeams[0];
 			}
 
-			actors.push_back( pollMoveKeysLock(cx, lock) );
-			actors.push_back(
+			actors.push_back(pollMoveKeysLock(cx, lock));
+			dataDistributionTrackerFuture =
 			    reportErrorsExcept(dataDistributionTracker(initData, cx, output, shardsAffectedByTeamFailure,
 			                                               getShardMetrics, getAverageShardBytes.getFuture(),
 			                                               readyToStart, anyZeroHealthyTeams, self->ddId, &shards),
-			                       "DDTracker", self->ddId, &normalDDQueueErrors()));
-			actors.push_back( reportErrorsExcept( dataDistributionQueue( cx, output, input.getFuture(), getShardMetrics, processingUnhealthy, tcis, shardsAffectedByTeamFailure, lock, getAverageShardBytes, self->ddId, storageTeamSize, configuration.storageTeamSize, &lastLimited ), "DDQueue", self->ddId, &normalDDQueueErrors() ) );
+			                       "DDTracker", self->ddId, &normalDDQueueErrors());
+			actors.push_back(dataDistributionTrackerFuture);
```

Comment:
It isn't necessary to add this future to `actors`, because it already has larger scope.

Reply:
Adding it to actors means that we're still waiting on it in the `wait(waitForAll(actors));` later on